### ISSUE TITLE
make `cview` module importable

### ISSUE THREAD

**Issue by utkarshayachit** [cinemascience/cinemasci#9]
Can we move the code under [1](https://github.com/cinemascience/cinemasci/blob/8e22839a266f391910a4546649c565b3107834c6/cinemasci/cview/__main__.py#L98) to a callable function, say `main(args)` and then call that? That way, another module that imports this module can still choose to execute the main-loop. 

The need for this change is as follows:

In ParaView, this package will be installed under `./lib/python3.6/site-packages/paraview/tpl/cinemasci`. To run the app, users will need to run pvpython with a longer path to -m. I think I'll add a cinema_viewer.py under paraview/ that internally forwards to cview/ to make it easier for ParaView users.

**Comment by dhrogers** [comment]
Yes. We will have both __main__ and `def main(args)`, so it can be called with the function or through `python -m`

I think that will work.

**Comment by dhrogers** [comment]
I've updated the code, and now you can access this through both a function call and through `python -m cinemasci.cview`

To access the function:
`
import cinemasci

cinemasci.cview.run_server( "view", "data/test.cdb", 8200, None)
`
Note that the `python -m` approach provides a method to parse command line args, while the function call obviously does not. Otherwise, it works either way.

I need to update the tests, and consider if this should be something callable from one level up: `cinemasci.run_cinema_server()`, but otherwise it's integrated and working.

**Comment by dhrogers** [comment]
@utkarshayachit : one issue is that the module needs to know the full path to the install directory, so that the html and javascript files can be served. The relative path is above, but is there any way to determine the full path at runtime?

**Comment by utkarshayachit** [comment]
```
import pathlib
pathlib.Path(__file__).parent.absolute()
```
This [ref](https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory) should give you the path for the current `.py` file.
